source/task_manager/crawl_tasks.py

```diff
@@ -14,12 +14,22 @@
 from apps.crawl_space.settings import LANG_DETECT_PATH
 from apps.crawl_space.models import Crawl
 
+# TODO - pull out this hardcode search
+if os.path.exists('/home/vagrant/miniconda/envs/memex/bin/nutch'):
```

Comment:
This is currently only triggering for vagrant builds, so it shouldn't affect other code.